### DISCUSSION TITLE
Add old (and better) "Printers and Devices" to Explorer patcher

### DISCUSSION THREAD

**Discussion by opjose** [valinet/ExplorerPatcher#2695] [Ideas]
Can you add this to Explorer Patcher?

shell:::{A8A91A66-3A7D-4424-8D24-04E180695C7A}

**Comment by pyrates999** [comment]
To where in EP?

**Comment by opjose** [comment]
Yes.

Win11 makes it impossible to find that dialog but it is needed for some functions.

Thanks!

Sent from Mail for Windows

**Comment by pyrates999** [comment]
You can find it by opening control panel.

**Comment by opjose** [comment]
No it does not appear in 23h2.

To bring it up you have to use the shell command I indicated.

This is why it would be nice to add it to EP.

Thanks.

Sent from Mail for Windows

**Comment by pyrates999** [comment]
https://www.youtube.com/watch?v=toaO2trSsgQ

**Comment by opjose** [comment]
I do know how to get to the old control panel thanks but that is not the point of this.


The old “devices and printers” screen is NOT available under Windows 11 22h2 and beyond as it has been replaced by the settings app pages.

In other words you cannot get to this specific screen: (See image)

The shell command I gave does open it.
 
  
![2024-01-14_00-34-06](https://github.com/valinet/ExplorerPatcher/assets/15528266/b03145c0-0f6d-4a8b-ac38-0910bcf5246e)

**Comment by pyrates999** [comment]
Ok, so what should this setting do?  Please provide the list of steps where you should end up in the old “devices and printers” instead of the one in settings.

My easiest method would be to install open shell and set it to the windows 7 style start menu and set control panel to be a menu.  That way you can easily get to it.

**Comment by opjose** [comment]
I think the easiest option is to prevent the redirection of the old control panel “devices and printers” icon to the modern app in the same way you already have “System / About Page” and the other options under the “other” page of Explorer Patcher.

That is a logical addition to what you have already done. The end user can then select the behavior they want along with the others.

Thanks!

Sent from Mail for Windows

**Comment by pyrates999** [comment]
Ok, thank you for the suggestion.  We'll see if it gets implemented.

**Comment by tallman69** [comment]
@opjose 
If you are not already aware of the information in this article, you may find this helpful:

https://www.elevenforum.com/t/create-devices-and-printers-shortcut-in-windows-11.7782/

**Comment by opjose** [comment]
I don’t know why people believe I am just talking about the old control panel.In windows click on start and run.Then paste the shell address I gave in the first post.Then try to get to the same screen that appears when you do that under windows 11 22h2 or greater.Then you will understand.Thanks Sent from a fancy digital device...On Jan 14, 2024, at 1:49 PM, tallman69 ***@***.***> wrote:﻿
@opjose
If you are not already aware of the information in this article, this has some helpful information:
https://www.howtogeek.com/761291/how-to-open-control-panel-on-windows-11/

**Comment by opjose** [comment]
Please look at this to help you understand…Redirectinganswers.microsoft.comSent from a fancy digital device...On Jan 14, 2024, at 1:49 PM, tallman69 ***@***.***> wrote:﻿
@opjose
If you are not already aware of the information in this article, this has some helpful information:
https://www.howtogeek.com/761291/how-to-open-control-panel-on-windows-11/

**Comment by tallman69** [comment]
@opjose 
My apologies. I edited my post before I saw your replies because I realized I had misunderstood what you were saying, I replaced the How-to Geek link with a link to a tutorial that explains how to create a shortcut to Devices and Printers.

https://www.elevenforum.com/t/create-devices-and-printers-shortcut-in-windows-11.7782/

I thought having this shortcut might make it easier for some to get to Devices and Printers since MS has, as you have pointed out, redirected that link in the Control Panel in Windows 11.  I myself did not realize MS had done that since I have been installing Open-Shell on my Windows 11 installs, and Open-Shell directs the Control Panel to still open Devices and Printers instead of the Settings app.

**Comment by opjose** [comment]
I appreciate your help.

I am aware of how to create a desktop link to the shell shortcut.

I posted about this because I found Explorer Patcher useful and adding the “Devices and Printers” redirection block logically follows what is already in the program.

Indeed it would fit perfectly as a new item in the “Prevent Control Panel Links from being redirected to the Settings App” in Explorer patcher.

Given that this has already been done for the other redirects I thought adding the new entry would be somewhat easy.

Hopefully that is true.

Thanks again!

**Comment by joaquincabrerasimoes** [comment]
I actually came here tu suggest the same thing.
It's basically adding to [Other→Prevent the following Control Panel links from being redirected to the Settings app]
The option [Devices and Printers]
So that to get to this point https://github.com/valinet/ExplorerPatcher/assets/15528266/b03145c0-0f6d-4a8b-ac38-0910bcf5246e you don't have to go
[Control Panel → Hardware and Sound → Devices and Printers → Settings(Bluetooth and devices) → Settings(Devices) → Settings(More devices and printer settings)]
You can just to [Control Panel → Hardware and Sound → Devices and Printers], like it was up until Windows 10

In the meantime, just in case you don't know @[opjose] there is a way.
Instead of [Control Panel → Hardware and Sound → Devices and Printers] using the big icons, use the little arrow in the address bar of the Control Panel

https://github.com/valinet/ExplorerPatcher/assets/34731493/da734f97-21ce-40b0-8095-edac72332679

But it would be awesome if this gets implemented.

**Comment by opjose** [comment]
Thanks!

I didn’t know that. Nice.

Learn something new everyday!


Sent from Mail for Windows
